ci/PIPELINE.md

```diff
@@ -0,0 +1,117 @@
+## Install and configure a Tekton pipeline on Openshift
+
+1) Install Red Hat OpenShift Pipelines latest (1.9.2 at the time of this doc)  from the Operator Hub
```

Comment:
Is there a official doc to link here?

Reply:
IS available this but is tied to a specific version of openshift https://docs.openshift.com/container-platform/4.12/cicd/pipelines/installing-pipelines.html